src/Response/DTO/Address.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\OverseasBundle\Response\DTO;
+
+class Address
+{
+    private ?string $addressId = null;
+    private string $name;
+    private string $zipCode;
+    private string $place;
+    private string $street;
+    private ?int $houseNumber = null;
```

Comment:
`?int` sounds dangerous, is it really always an int?

Reply:
The documentation says so. In our production json file, all numbers are ints as well 🤷🏻

Reply:
i now why it is like that. Because we have houseNumberAddition where is string like `A`